.github/workflows/multiarch-build.yaml

```
@@ -0,0 +1,35 @@
+name: Multiarch build
+
+on: [push]
+
+jobs:
+  multiarch-build:
+    name: Build images
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)